cypress/integration/workflow-executions.spec.js

```diff
@@ -5,77 +5,177 @@ import workflowsFixture from '../fixtures/workflows.json';
 const workflowRunningFixture = workflowsFixture.executions[0];
 const { workflowId, runId } = workflowRunningFixture.execution;
 
-const statuses = [
-  'Running',
-  'TimedOut',
-  'Completed',
-  'Failed',
-  'ContinuedAsNew',
-  'Canceled',
-  'Terminated',
-];
-
 describe('Workflow Executions List', () => {
   beforeEach(() => {
     cy.interceptApi();
 
     cy.visit('/namespaces/default/workflows');
 
     cy.wait('@workflows-api');
+    cy.wait('@workflows-count-api');
     cy.wait('@namespaces-api');
   });
 
+  it('should show count of workflows', () => {
+    cy.get('[data-cy="workflow-count"]').should('contain', '15 workflows');
+  });
+
   it('should default to All for the time range', () => {
-    cy.get('#time-range-filter').find('option').should('have.value', 'null');
+    cy.get('#time-range-filter').should('have.value', '');
   });
 
-  it('should default to showing all workflows', () => {
-    cy.get('#execution-status-filter')
-      .find('option:selected')
-      .should('have.value', 'null');
+  describe('Workflow Manual Search', () => {
+    it('should change url on manual search and update filters and show results count', () => {
+      cy.get('#manual-search').type('WorkflowType="ImportantWorkflowType"');
+      cy.get('[data-cy="manual-search-button"]').click();
+
+      const result = encodeURIComponent('WorkflowType="ImportantWorkflowType"');
+      cy.url().should('contain', result);
 
-    cy.get('#workflow-id-filter').should('have.value', '');
+      cy.get('[data-cy="workflow-type-filter-button"]').click();
+      cy.get('#workflowType').should('have.value', 'ImportantWorkflowType');
 
-    cy.get('#workflow-type-filter').should('have.value', '');
+      cy.get('[data-cy="workflow-count"]').should(
+        'contain',
+        'Results 15 of 15 workflows',
+      );
+    });
   });
 
   describe('Workflow Filters', () => {
-    it('should send the correct query for Workflow Type', () => {
-      const result = encodeURIComponent('WorkflowType="ImportantWorkflowType"');
+    it('should send the correct query for Workflow Type, autocomplete manual search and be clearable', () => {
+      cy.get('[data-cy="workflow-type-filter-button"]').click();
 
-      cy.get('#workflow-type-filter').type('ImportantWorkflowType');
+      cy.get('#workflowType').type('ImportantWorkflowType');
 
+      const result = encodeURIComponent('WorkflowType="ImportantWorkflowType"');
       cy.url().should('contain', result);
+      cy.get('#manual-search').should(
+        'have.value',
+        'WorkflowType="ImportantWorkflowType"',
+      );
+
+      cy.get('[data-cy="workflow-count"]').should(
+        'contain',
+        'Results 15 of 15 workflows',
+      );
+
+      cy.get(
+        '.px-2 > .input-container > [data-cy="clear-input"] > svg',
```

Comment:
This `.px-2` selector seems like it would be real brittle.

Reply:
So brittle. I cringed adding it, thanks for keeping me honest.